spec/avram/model_spec.cr

```diff
@@ -33,7 +33,6 @@ describe Avram::Model do
     user = UserFactory.create
     post = PostFactory.create
 
-    user.id.should eq(post.id)
```

Comment:
The id incrementing does not get reset with transaction rollbacks so there's no way to do this check. This spec seems a bit pointless but I did the minimal change.

Reply:
🤔  lol... is this just assuming that `user.id` and `post.id` will both be `1` here?  Yeah, that doesn't seem very helpful.. did I do that? 😂